perun-core/src/main/java/cz/metacentrum/perun/core/impl/MembersManagerImpl.java

```diff
@@ -979,4 +979,21 @@ private String getGroupStatusSQLConditionForMembersPage(MembersPageQuery query,
 		}
 		return groupStatusesQueryString;
 	}
+
+	public void rejectGroupApplications(PerunSession session, Member member) {
```

Comment:
This method selects only applications that are NOT rejected or approved. Also, it removed the logins, which may be unwanted .

Reply:
Just please take a look what is the order in execution of nested queries inside updates. Otherwise, we might end up with the same result as all ELIXIR users being renamed to `Deleted User` 😉 👼🏼 #notMyFail :D